src/sisl/geom/nanoribbon.py

```diff
@@ -17,14 +17,17 @@
     "graphene_nanoribbon",
     "agnr",
     "zgnr",
+    "cgnr",
     "heteroribbon",
     "graphene_heteroribbon",
 ]
 
 
 @set_module("sisl.geom")
-def nanoribbon(width: int, bond: float, atoms, kind: str = "armchair"):
-    r"""Construction of a nanoribbon unit cell of type armchair or zigzag.
+def nanoribbon(
+    width: int, bond: float, atoms, kind: str = "armchair", index: (int, int) = (3, 1)
```

Comment:
Good suggestion. I like `chirality`. Alternatively, we could also simply accept `kind=(3,1)` etc, understood as a chiral ribbon.

Reply:
I also thought about that, but would there be any other nanoribbons with chirality? If so it could be dangerous. I am not against it, just if it could prove to be ambiguous.

Reply:
The chirality index `(n, m)` specifies a certain direction $\theta$ in the graphene lattice, but the pair is not unique for the angle alone. However, for a ribbon the index also sets a specific edge structure: `n` controls the number of zigzag segments for every `m` armchair ones.